testsuite/integration-tests/src/test/java/org/jboss/resteasy/test/client/ClientDynamicFeaturesTest.java

```diff
@@ -60,7 +60,7 @@
      */
     @Test
     public void testDynamicFeatures() throws Exception {
-        ResteasyProviderFactory factory =  ResteasyProviderFactory.getInstance();
+        ResteasyProviderFactory factory =  ResteasyProviderFactory.newInstance();
```

Comment:
Same comment as before, regarding the need to call _RegisterBuiltin.register(factory);_